site/book/01-getting-started/02-quickstart.md

```diff
@@ -59,7 +59,7 @@ instance, you can automatically search and replace all the occurrences of `app`
 name on resources in the package using path expressions:
 
 ```shell
-$ kpt fn eval --image gcr.io/kpt-fn/search-replace:v0.1 -- 'by-path=spec.**.app' 'put-value=my-nginx'
```

Comment:
it's better to introduce the long-form first with both --image and the container image. Then at a later point (Chapter 4), can introduce these as porcelain.

Reply:
Updated. Quickstart will have long form, chapter 2+ will use short form.